From 019573f6ac99d77e22386c6fc8761c76ecf0aae0 Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Tue, 25 Jun 2024 12:41:20 -0700
Subject: [PATCH] Docs/translation/ko_KR: Fixup a warning

Documentation/translations/ko_KR/index.rst:11: WARNING: toctree contains reference to nonexisting document 'translations/ko_KR/     }\\kerneldocEndKR'

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/translations/ko_KR/index.rst | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/Documentation/translations/ko_KR/index.rst b/Documentation/translations/ko_KR/index.rst
index 24718c71eacf..a20772f9d61c 100644
--- a/Documentation/translations/ko_KR/index.rst
+++ b/Documentation/translations/ko_KR/index.rst
@@ -14,4 +14,6 @@
    process/howto
    core-api/wrappers/memory-barriers.rst
 
-    }\kerneldocEndKR
+.. raw:: latex
+
+       }\kerneldocEndKR
-- 
2.39.2

